dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -59,7 +59,7 @@ cookie.same.site.none=Set the session cookie SameSite attribute value to None
 cookie.same.site.strict=Set the session cookie SameSite attribute value to Strict
 
 # The word Partitioned should not be translated
-cookie.partitioned=Include a Partitioned attribute in session cookies
+cookie.partitioned=Partiton the session cookie
```

Comment:
Changed due to the error: 
```junit.framework.AssertionFailedError: The label for attribute cookiePartitioned of type com.ibm.ws.session contains the banned word attribute in the label "Include a Partitioned attribute in session cookies".```

Reply:
@volosied  you have a spelling issue here `partitioned` not `partiton`